shared/src/summarize_result.js

```diff
@@ -89,11 +89,16 @@ async function setup() {
 
     const searchParams = new URLSearchParams(window.location.search);
 
-    // If there's no URL, get the currently active tab and default params
-    if (!searchParams.get('url')) {
+    if (!searchParams.get('summary_type')) {
```

Comment:
The changes in this file allow the popup to be initialized with _just_ a URL param (right click context open), otherwise not all parameters would've been properly set.